### PR TITLE
feat(`launch`): add `chainCreationFee` param

### LINKED ISSUE #537: Create `ChainCreationFee` param
Add a `ChainCreationFee` as param in the `launch` module

This param would be `EmptyCoins` by default

If a value is defined, a fee is transferred from the coordinator account and set to the... community pool?

### PR DESCRIPTION
**Pull request by giunatale**
Closes #537

Depends on #567 being merged first.

### CODE REVIEW

**Review by aljo242 — DISMISSED** [review]

File: testutil/sample/launch.go [hunk 1]
```diff
@@ -301,7 +303,11 @@ func GenesisHash() string {
 func LaunchParams() launch.Params {
 	maxLaunchTime := launch.DefaultMaxLaunchTime - rand.Int63n(10)
 	minLaunchTime := rand.Int63n(10) + launch.DefaultMinLaunchTime
-	return launch.NewParams(minLaunchTime, maxLaunchTime, launch.DefaultRevertDelay)
+
+	// assign random small amount of staking denom
+	chainCreationFee := sdk.NewCoins(sdk.NewInt64Coin("stake", rand.Int63n(100)+1))
```
Comment (lubtd) [inline testutil/sample/launch.go:308]: Maybe we can use a constant for `"stake"`

File: x/launch/keeper/msg_create_chain_test.go [hunk 2]
```diff
@@ -3,38 +3,129 @@ package keeper_test
...
-	// Create a coordinator
-	coordAddress := sample.Address()
-	msgCreateCoordinator := sample.MsgCreateCoordinator(coordAddress)
+	// Create coordinators
+	coordMap := make(map[string]uint64)
+	msgCreateCoordinator := sample.MsgCreateCoordinator(coordAddr1)
 	resCoord, err := ts.ProfileSrv.CreateCoordinator(ctx, &msgCreateCoordinator)
 	require.NoError(t, err)
-	coordID := resCoord.CoordinatorID
+	coordMap[coordAddr1] = resCoord.CoordinatorID
```
Comment (lubtd) [inline x/launch/keeper/msg_create_chain_test.go:74]: Could we use a `for` loop for this?

File: x/launch/keeper/msg_create_chain_test.go [hunk 3]
```diff
@@ -3,38 +3,129 @@ package keeper_test
...
 
-	// Create a campaign
-	msgCreateCampaign := sample.MsgCreateCampaign(coordAddress)
+	// Create a campaign for each valid coordinator
+	campMap := make(map[string]uint64)
+	msgCreateCampaign := sample.MsgCreateCampaign(coordAddr1)
 	resCampaign, err := ts.CampaignSrv.CreateCampaign(ctx, &msgCreateCampaign)
 	require.NoError(t, err)
-	campaignID := resCampaign.CampaignID
+	campMap[coordAddr1] = resCampaign.CampaignID
```
Comment (lubtd) [inline x/launch/keeper/msg_create_chain_test.go:97]: Same

File: x/launch/keeper/msg_create_chain_test.go (hunk 2 above)
Comment (giunatale) [reply x/launch/keeper/msg_create_chain_test.go:74]: done the same also for `msg_create_campaign_test.go`. I didn't use a for loop here for consistency wrt to that file, but with a for loop for both it objectively looks cleaner. I've also switched to `sample.MsgCreateCoordinator()` while before it was using a custom declaration.

File: testutil/sample/simulation.go [hunk 4]
```diff
@@ -9,7 +9,12 @@ import (
 	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
 )
 
-const simAccountsNb = 100
+const (
+	//BondDenom defines the bond denom used in testing
```
Comment (aljo242) [inline testutil/sample/simulation.go:13]:
```suggestion
	// BondDenom defines the bond denom used in testing
```

**Review by aljo242 — CHANGES_REQUESTED** [review]
Tiny lint fix required.  These simplifications to the tests are sweet !

**Review by aljo242 — APPROVED** [review]

**Review by lubtd — APPROVED** [review]

**Pull request merged**